src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -36,6 +41,34 @@
 GESTATIONAL_AGE = "gestational_age"
 
 
+class LBWSGDistribution(LBWSGDistribution_):
```

Comment:
These components are so we use the correct columns to do lookup for birth exposure.